{
  "uid" : "4768c5ccf1bf33f",
  "name" : "Verify values of the text in links, buttons in the Header",
  "fullName" : "tests.header_test.TestHeaderPageText#test_hp_02_01_verify_text_in_links_and_buttons",
  "historyId" : "899ce41ff2f5a87d2f9030a44834068c",
  "time" : {
    "start" : 1728784158098,
    "stop" : 1728784158098,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5578453a802a <unknown>\n#1 0x55784508e5e0 <unknown>\n#2 0x5578450867eb <unknown>\n#3 0x557845076a09 <unknown>\n#4 0x557845078744 <unknown>\n#5 0x557845076cbd <unknown>\n#6 0x55784507652f <unknown>\n#7 0x557845076422 <unknown>\n#8 0x557845074493 <unknown>\n#9 0x557845074ada <unknown>\n#10 0x557845090f41 <unknown>\n#11 0x5578451228c5 <unknown>\n#12 0x557845102b22 <unknown>\n#13 0x557845121d7d <unknown>\n#14 0x5578451028c3 <unknown>\n#15 0x5578450d06b3 <unknown>\n#16 0x5578450d168e <unknown>\n#17 0x557845372a2b <unknown>\n#18 0x5578453769b1 <unknown>\n#19 0x55784535f225 <unknown>\n#20 0x557845377532 <unknown>\n#21 0x55784534438f <unknown>\n#22 0x557845396f28 <unknown>\n#23 0x5578453970f3 <unknown>\n#24 0x5578453a6e7c <unknown>\n#25 0x7fdca1494ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"bb9bd19ea7350163a88dea211a1b5358\")>\n\n    @pytest.fixture()\n    @allure.step(f'Open page: {MainPageLinks.URL_MAIN_PAGE}')\n    def main_page_open(driver):\n>       driver.get(MainPageLinks.URL_MAIN_PAGE)\n\ntests/conftest.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7faf1f7559a0>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x5578453970f3 \\\\u003Cunknown>\\\\n#24 0x5578453a6e7c \\\\u003Cunknown>\\\\n#25 0x7fdca1494ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=129.0.6668.89)\nE       Stacktrace:\nE       #0 0x5578453a802a <unknown>\nE       #1 0x55784508e5e0 <unknown>\nE       #2 0x5578450867eb <unknown>\nE       #3 0x557845076a09 <unknown>\nE       #4 0x557845078744 <unknown>\nE       #5 0x557845076cbd <unknown>\nE       #6 0x55784507652f <unknown>\nE       #7 0x557845076422 <unknown>\nE       #8 0x557845074493 <unknown>\nE       #9 0x557845074ada <unknown>\nE       #10 0x557845090f41 <unknown>\nE       #11 0x5578451228c5 <unknown>\nE       #12 0x557845102b22 <unknown>\nE       #13 0x557845121d7d <unknown>\nE       #14 0x5578451028c3 <unknown>\nE       #15 0x5578450d06b3 <unknown>\nE       #16 0x5578450d168e <unknown>\nE       #17 0x557845372a2b <unknown>\nE       #18 0x5578453769b1 <unknown>\nE       #19 0x55784535f225 <unknown>\nE       #20 0x557845377532 <unknown>\nE       #21 0x55784534438f <unknown>\nE       #22 0x557845396f28 <unknown>\nE       #23 0x5578453970f3 <unknown>\nE       #24 0x5578453a6e7c <unknown>\nE       #25 0x7fdca1494ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1728784074935,
      "stop" : 1728784074936,
      "duration" : 1
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "main_page_open",
    "time" : {
      "start" : 1728784158869,
      "stop" : 1728784159298,
      "duration" : 429
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5578453a802a <unknown>\n#1 0x55784508e5e0 <unknown>\n#2 0x5578450867eb <unknown>\n#3 0x557845076a09 <unknown>\n#4 0x557845078744 <unknown>\n#5 0x557845076cbd <unknown>\n#6 0x55784507652f <unknown>\n#7 0x557845076422 <unknown>\n#8 0x557845074493 <unknown>\n#9 0x557845074ada <unknown>\n#10 0x557845090f41 <unknown>\n#11 0x5578451228c5 <unknown>\n#12 0x557845102b22 <unknown>\n#13 0x557845121d7d <unknown>\n#14 0x5578451028c3 <unknown>\n#15 0x5578450d06b3 <unknown>\n#16 0x5578450d168e <unknown>\n#17 0x557845372a2b <unknown>\n#18 0x5578453769b1 <unknown>\n#19 0x55784535f225 <unknown>\n#20 0x557845377532 <unknown>\n#21 0x55784534438f <unknown>\n#22 0x557845396f28 <unknown>\n#23 0x5578453970f3 <unknown>\n#24 0x5578453a6e7c <unknown>\n#25 0x7fdca1494ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/",
      "time" : {
        "start" : 1728784158870,
        "stop" : 1728784159298,
        "duration" : 428
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5578453a802a <unknown>\n#1 0x55784508e5e0 <unknown>\n#2 0x5578450867eb <unknown>\n#3 0x557845076a09 <unknown>\n#4 0x557845078744 <unknown>\n#5 0x557845076cbd <unknown>\n#6 0x55784507652f <unknown>\n#7 0x557845076422 <unknown>\n#8 0x557845074493 <unknown>\n#9 0x557845074ada <unknown>\n#10 0x557845090f41 <unknown>\n#11 0x5578451228c5 <unknown>\n#12 0x557845102b22 <unknown>\n#13 0x557845121d7d <unknown>\n#14 0x5578451028c3 <unknown>\n#15 0x5578450d06b3 <unknown>\n#16 0x5578450d168e <unknown>\n#17 0x557845372a2b <unknown>\n#18 0x5578453769b1 <unknown>\n#19 0x55784535f225 <unknown>\n#20 0x557845377532 <unknown>\n#21 0x55784534438f <unknown>\n#22 0x557845396f28 <unknown>\n#23 0x5578453970f3 <unknown>\n#24 0x5578453a6e7c <unknown>\n#25 0x7fdca1494ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"bb9bd19ea7350163a88dea211a1b5358\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  }, {
    "name" : "driver",
    "time" : {
      "start" : 1728784158099,
      "stop" : 1728784158869,
      "duration" : 770
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1728784159330,
      "stop" : 1728784159393,
      "duration" : 63
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Test Header"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "header_test"
  }, {
    "name" : "subSuite",
    "value" : "TestHeaderPage"
  }, {
    "name" : "host",
    "value" : "fv-az891-551"
  }, {
    "name" : "thread",
    "value" : "2367-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.header_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 3,
        "skipped" : 0,
        "passed" : 28,
        "unknown" : 0,
        "total" : 31
      },
      "items" : [ {
        "uid" : "56804c7e5659b3b3",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/60//#testresult/56804c7e5659b3b3",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55a8d5f7c02a <unknown>\n#1 0x55a8d5c625e0 <unknown>\n#2 0x55a8d5c5a7eb <unknown>\n#3 0x55a8d5c4aa09 <unknown>\n#4 0x55a8d5c4c744 <unknown>\n#5 0x55a8d5c4acbd <unknown>\n#6 0x55a8d5c4a52f <unknown>\n#7 0x55a8d5c4a422 <unknown>\n#8 0x55a8d5c48493 <unknown>\n#9 0x55a8d5c48ada <unknown>\n#10 0x55a8d5c64f41 <unknown>\n#11 0x55a8d5cf68c5 <unknown>\n#12 0x55a8d5cd6b22 <unknown>\n#13 0x55a8d5cf5d7d <unknown>\n#14 0x55a8d5cd68c3 <unknown>\n#15 0x55a8d5ca46b3 <unknown>\n#16 0x55a8d5ca568e <unknown>\n#17 0x55a8d5f46a2b <unknown>\n#18 0x55a8d5f4a9b1 <unknown>\n#19 0x55a8d5f33225 <unknown>\n#20 0x55a8d5f4b532 <unknown>\n#21 0x55a8d5f1838f <unknown>\n#22 0x55a8d5f6af28 <unknown>\n#23 0x55a8d5f6b0f3 <unknown>\n#24 0x55a8d5f7ae7c <unknown>\n#25 0x7f6986094ac3 <unknown>",
        "time" : {
          "start" : 1728697272069,
          "stop" : 1728697272069,
          "duration" : 0
        }
      }, {
        "uid" : "ad00f8235bb688a1",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/59//#testresult/ad00f8235bb688a1",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55ae4df1402a <unknown>\n#1 0x55ae4dbfa5e0 <unknown>\n#2 0x55ae4dbf27eb <unknown>\n#3 0x55ae4dbe2a09 <unknown>\n#4 0x55ae4dbe4744 <unknown>\n#5 0x55ae4dbe2cbd <unknown>\n#6 0x55ae4dbe252f <unknown>\n#7 0x55ae4dbe2422 <unknown>\n#8 0x55ae4dbe0493 <unknown>\n#9 0x55ae4dbe0ada <unknown>\n#10 0x55ae4dbfcf41 <unknown>\n#11 0x55ae4dc8e8c5 <unknown>\n#12 0x55ae4dc6eb22 <unknown>\n#13 0x55ae4dc8dd7d <unknown>\n#14 0x55ae4dc6e8c3 <unknown>\n#15 0x55ae4dc3c6b3 <unknown>\n#16 0x55ae4dc3d68e <unknown>\n#17 0x55ae4dedea2b <unknown>\n#18 0x55ae4dee29b1 <unknown>\n#19 0x55ae4decb225 <unknown>\n#20 0x55ae4dee3532 <unknown>\n#21 0x55ae4deb038f <unknown>\n#22 0x55ae4df02f28 <unknown>\n#23 0x55ae4df030f3 <unknown>\n#24 0x55ae4df12e7c <unknown>\n#25 0x7fe28a694ac3 <unknown>",
        "time" : {
          "start" : 1728610963398,
          "stop" : 1728610963398,
          "duration" : 0
        }
      }, {
        "uid" : "560d6bd24a196a38",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/58//#testresult/560d6bd24a196a38",
        "status" : "passed",
        "time" : {
          "start" : 1728524766021,
          "stop" : 1728524766210,
          "duration" : 189
        }
      }, {
        "uid" : "ced638f216c9bbfe",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/57//#testresult/ced638f216c9bbfe",
        "status" : "passed",
        "time" : {
          "start" : 1728438442276,
          "stop" : 1728438442466,
          "duration" : 190
        }
      }, {
        "uid" : "4d29d1c6e3e6475",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/56//#testresult/4d29d1c6e3e6475",
        "status" : "passed",
        "time" : {
          "start" : 1728352010699,
          "stop" : 1728352010921,
          "duration" : 222
        }
      }, {
        "uid" : "c190bb6ee8581dfc",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/55//#testresult/c190bb6ee8581dfc",
        "status" : "passed",
        "time" : {
          "start" : 1728266111378,
          "stop" : 1728266111570,
          "duration" : 192
        }
      }, {
        "uid" : "82e817665487dccb",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/54//#testresult/82e817665487dccb",
        "status" : "passed",
        "time" : {
          "start" : 1728179558704,
          "stop" : 1728179558904,
          "duration" : 200
        }
      }, {
        "uid" : "47a07085718d1126",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/53//#testresult/47a07085718d1126",
        "status" : "passed",
        "time" : {
          "start" : 1728092759565,
          "stop" : 1728092759752,
          "duration" : 187
        }
      }, {
        "uid" : "d393e840720f721a",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/52//#testresult/d393e840720f721a",
        "status" : "passed",
        "time" : {
          "start" : 1728006462784,
          "stop" : 1728006462969,
          "duration" : 185
        }
      }, {
        "uid" : "de26852a7fa4227b",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/51//#testresult/de26852a7fa4227b",
        "status" : "passed",
        "time" : {
          "start" : 1727920090994,
          "stop" : 1727920091192,
          "duration" : 198
        }
      }, {
        "uid" : "1359228044ff350f",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/50//#testresult/1359228044ff350f",
        "status" : "passed",
        "time" : {
          "start" : 1727833559133,
          "stop" : 1727833559322,
          "duration" : 189
        }
      }, {
        "uid" : "3e478be2f174d595",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/49//#testresult/3e478be2f174d595",
        "status" : "passed",
        "time" : {
          "start" : 1727747696374,
          "stop" : 1727747696582,
          "duration" : 208
        }
      }, {
        "uid" : "4f14fc3b8de7bc05",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/48//#testresult/4f14fc3b8de7bc05",
        "status" : "passed",
        "time" : {
          "start" : 1727660981750,
          "stop" : 1727660981962,
          "duration" : 212
        }
      }, {
        "uid" : "a98632aba4d766f0",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/47//#testresult/a98632aba4d766f0",
        "status" : "passed",
        "time" : {
          "start" : 1727574792852,
          "stop" : 1727574793066,
          "duration" : 214
        }
      }, {
        "uid" : "57bf1ff96a915187",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/46//#testresult/57bf1ff96a915187",
        "status" : "passed",
        "time" : {
          "start" : 1727488031213,
          "stop" : 1727488031398,
          "duration" : 185
        }
      }, {
        "uid" : "35303a64bf7493f",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/45//#testresult/35303a64bf7493f",
        "status" : "passed",
        "time" : {
          "start" : 1727401710725,
          "stop" : 1727401710915,
          "duration" : 190
        }
      }, {
        "uid" : "9711b71114bf7a5f",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/44//#testresult/9711b71114bf7a5f",
        "status" : "passed",
        "time" : {
          "start" : 1727315199409,
          "stop" : 1727315199602,
          "duration" : 193
        }
      }, {
        "uid" : "1c1bd86cec551e16",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/43//#testresult/1c1bd86cec551e16",
        "status" : "passed",
        "time" : {
          "start" : 1727228865169,
          "stop" : 1727228865354,
          "duration" : 185
        }
      }, {
        "uid" : "3c3e6cd3e7ff7a40",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/42//#testresult/3c3e6cd3e7ff7a40",
        "status" : "passed",
        "time" : {
          "start" : 1727142415623,
          "stop" : 1727142415838,
          "duration" : 215
        }
      }, {
        "uid" : "9380e5e55cae9adc",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/41//#testresult/9380e5e55cae9adc",
        "status" : "passed",
        "time" : {
          "start" : 1727055952316,
          "stop" : 1727055952496,
          "duration" : 180
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "4768c5ccf1bf33f.json",
  "parameterValues" : [ ]
}